docs/pages/access-controls/guides/webauthn.mdx

```diff
@@ -41,9 +41,13 @@ Teleport configuration as below:
         # To enable WebAuthn support, set this field to 'on', 'optional' or 'webauthn'
         second_factor: on
         webauthn:
+          # Required, replace with proxy web address (example.com, mytenant.teleport.sh),
+          #  see `webauthn` Fields Definitions for details.
```

Comment:
This comment applies to all the changes.

My concern is that once this is copied into a config, the reference to a section in this page is lost. For these comments to be useful once in place, they would have to have some information themselves, or link to the docs page. 
But if that's not a concern to anyone else, my remaining comment is to remove the backticks, since they do nothing in this format.

Reply:
I agree with the concern. Let's just put a very short explanation in the YAML rather than mention another part of the doc. For example:

```
   # Optionally allow specific MFA device vendors.
   attestation_allowed_cas:
```

Reply:
See updates